{{cookiecutter.project_slug}}/CONTRIBUTING.rst

```diff
@@ -9,96 +9,128 @@ Contributing
-official {{ cookiecutter.project_name }} docs, in docstrings, or even on the web in blog posts,
+{{ cookiecutter.project_slug }} could always use more documentation, whether as part of the
+official {{ cookiecutter.project_slug }} docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
 Get Started!
 ------------
 
-Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development.
+Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development using the command-line interface. Note that several alternative user interfaces exist, e.g., the Git GUI, `GitHub Desktop <https://desktop.github.com/>`_, extensions in `Visual Studio Code <https://code.visualstudio.com/>`_ ...
 
-1. Fork the `{{ cookiecutter.project_slug }}` repo on {% if rwth %}GitLab{% elif github %}GitHub{% endif %}.
-2. Clone your fork locally::
+1. `Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo/>`_ the `{{ cookiecutter.project_slug }}` repo on GitHub.
+2. Clone your fork locally and cd into the {{ cookiecutter.project_slug }} directory::
 
-    $ git clone {{ cookiecutter.git_service }}/{{ cookiecutter.git_username }}/{{ cookiecutter.project_slug }}.git
+    $ git clone https://github.com/YOUR_USERNAME/{{ cookiecutter.project_slug }}.git
+    $ cd {{ cookiecutter.project_slug }}
 
-3. Install your local copy into a virtual environment. Assuming you have Anaconda or Miniconda installed, this is how you set up your fork for local development::
+3. Install your local copy into a virtualenv. Assuming you have Anaconda or Miniconda installed, this is how you set up your fork for local development::
 
-    $ conda create --name {{ cookiecutter.project_slug }} python pip
+    $ conda create --name {{ cookiecutter.project_slug }} python
     $ conda activate {{ cookiecutter.project_slug }}
-    $ cd {{ cookiecutter.project_slug }}
-    $ pip install -r requirements_dev.txt
+    $ conda install pip
     $ pip install -e .
+    $ pip install -r requirements_dev.txt
 
-4. Create a branch for local development::
+4. Create a branch for local development. Indicate the intention of your branch in its respective name (i.e. `feature/branch-name` or `bugfix/branch-name`)::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
    Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+   tests::
 
     $ flake8 {{ cookiecutter.project_slug }} tests
     $ pytest
 
-   To get flake8 and tox, just pip install them into your virtualenv.
+   flake8 test must pass without any warnings for `./{{ cookiecutter.project_slug }}` and `./tests` using the default or a stricter configuration. Flake8 ignores `E123/E133, E226` and `E241/E242` by default. If necessary adjust the your flake8 and linting configuration in your IDE accordingly.
```

Comment:
```suggestion
   flake8 test must pass without any warnings for `./{{ cookiecutter.project_slug }}` and `./tests` using the default or a stricter configuration. Flake8 ignores `E123/E133, E226` and `E241/E242` by default. If necessary adjust your flake8 and linting configuration in your IDE accordingly.
```

Reply:
this is copied from pyfar, we should first discuss to change it there. then we can aply it here.

Reply:
added pr in pyfar for this, good that you noticed it
https://github.com/pyfar/pyfar/pull/545